ckanext/recombinant/views.py

```diff
@@ -116,15 +121,13 @@ def delete_records(id: str, resource_id: str) -> Union[str, Response]:
     dataset = lc.action.recombinant_show(
         dataset_type=pkg['type'], owner_org=org['name'])
 
-    def delete_error(err: str) -> str:
-        return render('recombinant/resource_edit.html',
-                      extra_vars={'delete_errors': [err],
-                                  'dataset': dataset,
-                                  'dataset_type': dataset['dataset_type'],
-                                  'resource': res,
-                                  'organization': org,
-                                  'filters': filters,
-                                  'action': 'edit'})
+    def delete_error(err: Union[str, Markup]) -> Union[str, Markup]:
+        session['RECOMBINANT_DELETE_ERRORS'] = err
+        session['RECOMBINANT_FILTERS'] = filters
```

Comment:
it's a little unfortunate to have to rely on session data (I know we're already doing this with the flash message). It's nicer to render the page again with the error shown instead of navigating away first.

Reply:
Yeah, the Recombinant views were really made pre 2.10. The flask view only now does not render things the same way :/ 

Because most of these views are POST only, there is not an easy way to just return self with errors. That being said, I have added `GET` to the records delete view here and reworked the view a bit to allow for that.